package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "lint": "eslint test",
+    "lint": "eslint index.js test/**/*-test.js",
     "test": "mocha test/**/*-test.js",
```

Comment:
Our CI misses linting functionality. Could you please modify the "test" task to run the linter before actual tests?